python/cudf_polars/cudf_polars/experimental/benchmarks/pdsh.py

```diff
@@ -1014,39 +1014,50 @@ def _query_type(query: int | str) -> list[int]:
 parser = argparse.ArgumentParser(
     prog="Cudf-Polars PDS-H Benchmarks",
     description="Experimental streaming-executor benchmarks.",
+    formatter_class=argparse.RawTextHelpFormatter,
 )
 parser.add_argument(
     "query",
     type=_query_type,
-    help="Query number.",
+    help=(
+        "Query to run. One of the following:\n"
+        "  - A single number (e.g., 11)\n"
+        "  - A comma-separated list of query numbers (e.g., 1,3,7)\n"
+        "  - The string 'all' to run all queries (1 through 22)"
+    ),
```

Comment:
If you want to avoid the `\n`, this should be the same.

```suggestion
    help=textwrap.dedent("""\
        Query to run. One of the following:
          - A single number (e.g., 11)
          - A comma-separated list of query numbers (e.g., 1,3,7)
          - The string 'all' to run all queries (1 through 22)
    ),
```